Add callback on tileset finished loading

miatang13 opened the pull request:
- Depends on this PR from Cesium Native: https://github.com/CesiumGS/cesium-native/pull/525 
- Solves #774  
- This PR contains adding a new delegate `OnTilesetLoaded` that is exposed to the blueprint, which is invoked when the current tileset has finished loading 
- Currently we determine a tileset to be loaded when there are no tiles to be loaded in the queues, no tiles currently in load, and no occlusion results that we are waiting on.

cesium-concierge commented:
Thanks for the pull request @miatang13!

* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] [Cesium for Unreal Samples](https://github.com/CesiumGS/cesium-unreal-samples) works.

kring commented:
@nithinp7 since you merged the cesium-native side of this, can you please review this PR as well?

nithinp7 commented:
Thanks @miatang13! Finally got a chance to try this out, it's super cool and the callback works well, even from BP. My only suggestion for this one would be to expose a (BP accessible) function on `Cesium3DTileset` to get the load progress as a percentage. 

Also related to that, the current load progress is being cast to an unsigned int, I think it might be better to leave it as a float what do you think? I should have caught that before merging the Cesium Native PR, since that would require slightly tweaking the Cesium Native api, but I think it would be worth it.

nithinp7 commented:
Also please update the changelog here.

miatang13 commented:
This PR currently depends on this PR from Cesium Native: https://github.com/CesiumGS/cesium-native/pull/533 

I have added a new blueprint getter function that retrieves the current loaded progress. Also I've updated the changelog. @nithinp7

nithinp7 commented:
Perfect, looks good to me!

DSoftwareArtist commented:
I assumed that this had been implemented. May I know how this feature works? Is there any example? @nithinp7
